lib/patch-series.ts

```diff
@@ -819,7 +819,7 @@ export class PatchSeries {
 
         if (publishTagsAndNotesToRemote) {
             logger.log("Publishing tag");
-            await git(["push", publishTagsAndNotesToRemote, this.notes.notesRef, `refs/tags/${tagName}`], {
+            await git(["push", publishTagsAndNotesToRemote, `refs/tags/${tagName}`], {
```

Comment:
I was wondering if this bit of code should be under the `dryRun` check just above it but it appears the only user of `dryRun` was removed in PR #1520.  Maybe `options.dryRun` is not needed any more?

Reply:
I am not sure that I follow... I mean, I do agree that the `dryRun` option was only useful for the command-line interface when running this locally (originally as [the `mail-patch-series.sh` shell script](https://github.com/dscho/mail-patch-series)).

However, even expanding the context prior to this hunk, I see:

```typescript
        if (!this.options.dryRun) {
            const key = this.metadata.pullRequestURL || this.project.branchName;
            await this.notes.set(key, this.metadata, true);
        }
```

As such, I think you're absolutely correct that this should be guarded by that option, which I will do.

Reply:
> this should be guarded by that option

Well, in the end I was really wondering if `options.dryRun` support can be removed.  Not sure if it is checked everywhere it should be (ie should `publishBranch` check it?).

That could be a separate pull request.

Reply:
> I was really wondering if `options.dryRun` support can be removed.

Me, too, and I am convinced that the answer is: Yes!

> That could be a separate pull request.

Totally!